perun-db/postgres.sql

```diff
@@ -1550,6 +1550,48 @@ create table groups_to_register (
     constraint grpreg_group_fk foreign key (group_id) references groups(id) on delete cascade
 );
 
+-- CONSENTS - consents of users
+
+create table consents (
+    id integer not null,
+    user_id integer not null,
+    consent_hub_id integer not null,
+	status integer not null default 0,
+	created_at timestamp default statement_timestamp() not null,
+	created_by varchar default user not null,
+	modified_at timestamp default statement_timestamp() not null,
+	modified_by varchar default user not null,
+	created_by_uid integer,
+	modified_by_uid integer,
+	constraint consents_pk primary key(id),
+	constraint consents_user_fk foreign key(user_id) references users(id),
+	constraint consents_cons_hub_fk foreign key(consent_hub_id) references consent_hubs(id)
+);
+
+-- CONSENT_ATTR_VALUES - values of attributes assigned to users
+create table consent_attr_values (
+	consent_id integer not null,  --identifier of consent (consents.id)
+	attr_id integer not null,  --identifier of attribute (attr_names.id)
+	attr_value text,  --attribute value
+	created_at timestamp default statement_timestamp() not null,
+	created_by varchar default user not null,
+	modified_at timestamp default statement_timestamp() not null,
+	modified_by varchar default user not null,
+	created_by_uid integer,
+	modified_by_uid integer,
+	constraint consentav_pk primary key(consent_id, attr_id),
+	constraint consentav_consent_fk foreign key (consent_id) references consents(id),
+  	constraint consentav_nam_fk foreign key (attr_id) references attr_names(id)
+);
+
+-- CONSENT_ATTR_U_VALUE - unique attribute values
+CREATE TABLE consent_attr_u_values (
```

Comment:
As we want to keep information about attribute definition that user gave consent to, and (for now) we don't need to store value of the attribute, this table should contain reference to `attr_names` id (for attribute definition) and `consent_id`. No need to store its value and table `consent_attr_values` should be removed completely. You can rename this one then. I think we can even omit the created/modified at/by values because they are relevant from the `consents` table.